CONTRIBUTING.md

```diff
@@ -71,6 +71,7 @@ To add a pre commit hook:
 
 - Build `google-java-format` jar following instructions from [github page](https://github.com/google/google-java-format).
 - Copy `config/git-hooks/pre-commit.template` to `.git/hooks/pre-commit`
+- Make the script executable `chmod +x .git/hooks/pre-commit`
```

Comment:
Could you change it to (add a colon after "executable"):

Make the script executable:  `chmod +x .git/hooks/pre-commit`